pkg/manager/member/tikv_member_manager_test.go

```diff
@@ -2196,6 +2196,43 @@ func TestTransformTiKVConfigMap(t *testing.T) {
 	}
 }
 
+func TestTransformTiKVBackupConfig(t *testing.T) {
+	g := NewGomegaWithT(t)
+	type testcase struct {
+		name       string
+		numThreads int64
+		result     string
+	}
+	tests := []testcase{
+		{
+			name:       "under 4.0",
+			numThreads: 24,
+			result: `[backup]
+  num-threads = 24
+`,
+		},
+		{
+			name:       "4.0.3",
+			numThreads: 24,
+			result: `[backup]
+  num-threads = 24
+`,
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			tc := newTidbClusterForTiKV()
+			tc.Spec.TiKV.Config.Backup = &v1alpha1.TiKVBackupConfig{
+				NumThreads: pointer.Int64Ptr(test.numThreads),
+			}
+			confText, err := MarshalTOML(tc.Spec.TiKV.Config)
+			g.Expect(err).NotTo(HaveOccurred())
+			g.Expect(test.result).Should(Equal(transformTiKVConfigMap(string(confText), tc)))
```

Comment:
`transformTiKVConfigMap` is used to fix an incompatible field in different TiKV versions. No need to call this function here. I would suggest removing this function and update this test to verify the marshaled toml string only.